### ISSUE TITLE
PKC/calcium-dependent protein serine/threonine kinase activity: 3 terms have the same meaning 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#25457]
GO:0004698	calcium-dependent protein kinase C activity
GO:0009931	calcium-dependent protein serine/threonine kinase activity
GO:0010857	calcium-dependent protein kinase activity

all mean the same thing. 

GO:0004698 has an EC mapping (2.7.11.1); others dont. 
The most accurate and informative label is 'calcium-dependent protein serine/threonine kinase activity'

Annotations are here: https://github.com/geneontology/go-annotation/issues/4597

2 of these terms should be obsoleted and replaced by the third one. 

Thanks, Pascale

**Comment by deustp01** [comment]
Are you saying that all calcium-dependent protein kinases are in fact specifically calcium-dependent protein serine/threonine kinases (GO:0009931), so annotations to GO:0004698 (we have 1) and GO:0010857  (we have 3) should be made to GO:0009931 instead? Don't know the details of protein kinase classes, so this is  sanity check to be sure that this is the biologically correct re-annotation.
Thanks

**Comment by raymond91125** [comment]
GO:0004698 calcium-dependent protein kinase C activity has XREF EC 2.7.11.13  (also XREF by GO:0004697, GO:0004699), which is protein kinase C that can be calcium dependent or independent. https://enzyme.expasy.org/EC/2.7.11.13
PKC has three categories, depending on their requirements for Ca2+, DAG, and phospholipid for activation. https://en.wikipedia.org/wiki/Protein_kinase_C. So perhaps calcium-dependent protein kinase C activity refers to "conventional - require DAG, Ca2+, and phospholipid for activation".
Furthermore, there are calcium/calmodulin-dependent protein (CaM) kinases in the annotation table. They are not PKC.

**Comment by pgaudet** [comment]
Right, thanks @raymond91125 , I truncated the EC - so this is what we have 


<img width="375" alt="image" src="https://github.com/geneontology/go-ontology/assets/4782928/61a9091f-1a4c-4feb-864e-9f660e8594d8">


I think these 5 terms are the same: 

GO:0004699 | calcium-independent protein kinase C   activity
GO:0004698 | calcium-dependent protein kinase C activity
GO:0004697 | protein kinase C activity
GO:0010857 | calcium-dependent protein kinase activity
GO:0009931 | calcium-dependent protein serine/threonine kinase activity

Thanks, Pascale

**Comment by pgaudet** [comment]
Thanks for pointing that out. It's a bit odd then - calcium-independent PKC then a serine/threonine protein kinase, right ? We would not group MFs by the protein family they belong to... 

@raymond91125 what are your thoughts on this?

**Comment by raymond91125** [comment]
@pgaudet Sorry, I'm not sure what the goal is now. If we merge the 5 terms above, what should we call it? PKC is not necessarily calcium dependent. Also, we are then mixing activities with different activating mechanisms together, is that desirable?

**Comment by raymond91125** [comment]
Some calcium-activated protein kinases may be of tyrosine PMID:26150074, distinguishing calcium-dependent protein kinase activity from calcium-dependent protein serine/threonine kinase activity. Thus it seems that all of 5 of these terms are distinct (or grouping for more than one child) and perhaps should not be merged.

**Comment by pgaudet** [comment]
Are you sure this is the same activity? This is what UniProt has for PTK2B (https://www.uniprot.org/uniprotkb/Q14289/entry)

> Activated in response to stimuli that lead to increased intracellular Ca2+ levels; this activation is indirect and may be mediated by calcium-mediated production of reactive oxygen species (ROS). 

And PTK2B is annotated to EC:2.7.10.2. 

Thanks, Pascale

**Comment by raymond91125** [comment]
"Although the mechanism of activation of Pyk2 is not known in detail, the FERM domain has been suggested to mediate Ca2+/CaM-dependent Pyk2 homodimerization (via binding of CaM to the α2-helix of the F2 subdomain) and transphosphorylation at Tyr402 to recruit Src, which binds via its SH2 domain and phosphorylates Pyk2 at Tyr579 and Tyr580 within the activation loop of the kinase domain to generate maximal Pyk2 activity (Kohno et al. [2008](https://link.springer.com/article/10.1007/s10974-015-9416-2#ref-CR23))."

**Comment by raymond91125** [comment]
My point is that if we merge these terms, we may have to reverse the merge, whereas not merging them does not seem to cause significant harm.

**Comment by pgaudet** [comment]
I'll bring this op on a signaling cassettes call. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Proposal: 

- [ ] GO:0004697 | protein kinase C activity:  rename 'diacylglycerol-dependent serine/threonine kinase activity'
- [ ] GO:0004698 | calcium-dependent protein kinase C activity: rename  'calcium,diacylglycerol-dependent serine/threonine kinase activity'
- [ ] GO:0004699 | calcium-independent protein kinase C activity >> obsolete, replace by 'GO:0004697 | protein kinase C activity' 
- [ ] GO:0010857 | calcium-dependent protein kinase activity   >> obsolete, consider GO:0004699 | calcium-independent protein kinase C activity 
- [ ] GO:0009931 | calcium-dependent protein serine/threonine kinase activity >> obsolete, consider GO:0004699 | calcium-independent protein kinase C activity 
- [ ] new term: 'calcium-dependent tyrosine kinase activity' PMID:26150074

**Comment by raymond91125** [comment]
The atypical PKCs do not depend on DAG. https://en.wikipedia.org/wiki/Protein_kinase_C, https://www.alliancegenome.org/gene/HGNC:9412

**Comment by raymond91125** [comment]
I suggest that we make things simple and less granular. 
1. Merge calcium-dependent protein kinase C activity and calcium-independent protein kinase C activity into protein kinase C activity, EC:2.7.11.13
2. The only IS_A parent for the solo protein kinase C activity is protein serine/threonine kinase activity. This follows the EC hierarchy, 2.7.11.-: Protein-serine/threonine kinases.
3. We can leave calcium-dependent protein kinase activity and calcium-dependent protein serine/threonine kinase activity intact.

**Comment by raymond91125** [comment]
Maybe I'm not getting the correct source but I thought EC:2.7.11.13 covers the atypical ones. https://www.ebi.ac.uk/intenz/query?cmd=SearchID&id=16301&view=INTENZ

**Comment by cmungall** [comment]
EC:2.7.11.13 is used to annotate the atypical ones, so as GO doesn't represent protein families the mapping between the GO term and EC has to be inexact to prevent incorrect propagation

**Comment by raymond91125** [comment]
CDPK1 https://amigo.geneontology.org/amigo/gene_product/UniProtKB:Q7XZK5 is not a PKC but a calcium-dependent, calmodulin-independent protein S/T kinase PMID:22760783

**Comment by pgaudet** [comment]
Hello again,

Looking at that once again, and I came to the same conclusion - all calcium-dependent PK are ser/thr. If we find a real example of Tyr kinase that is calcium-dependent, we can create this term as a child of tyrosine kinase. GO:0010857 calcium-dependent protein kinase activity is an unnecessary grouping term. 

I will obsolete GO:0010857 calcium-dependent protein kinase activity is an unnecessary grouping term & replace by its child 
GO.0009931 calcium-dependent protein serine/threonine kinase activity. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0010857 calcium-dependent protein kinase activity is an unnecessary grouping term & replace by its child GO.0009931 calcium-dependent protein serine/threonine kinase activity. 

There are no mappings; there are 8 EXP annotations to this term that will be automatically replaced, see 
https://github.com/geneontology/go-annotation/issues/5324

You can comment on the ticket: https://github.com/geneontology/go-ontology/issues/25457

Thanks, Pascale

**Comment by pgaudet** [comment]
Add do_not_annotate to [diacylglycerol-dependent serine/threonine kinase activity](http://purl.obolibrary.org/obo/GO_0004697)? 

Wait for reviews here: https://github.com/geneontology/go-annotation/issues/5327

**Comment by raymond91125** [comment]
Please re-assign this to me if needed. Thanks.
